index.html

```diff
@@ -20,34 +20,160 @@
+        </div>
+        <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card2 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card ard3 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card4 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card5 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card6 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about_container">
+      <h2 class="text_about">About me</h2>
+      <p class="text2_about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button type="button" id="button_resume"><p class="p_resume">Get My Resume</p></button>
+    </div>
+    <div class="skills_container">
+      <div class="card_skills">
+          <img src="./img/Icon Langguage.png" alt="" id="icons">
```

Comment:
- Just as the previous reviewer suggested, please name your image files in this format `lowercase_with_underscores.png` :+1: This is because spaces inside of a URL or a linked file basically generate a faulty syntax that the server reads as the end of a character string.  The server sees "the end" and stops processing and when the full string is not processed, there is a possibility that it would not be properly represented on your computer screen hence causing a broken link. So please let's solve this :wink: